site/es_MX.all.json

```diff
@@ -409,5 +409,6 @@
 
   "app_badge_title": { "other": "Download the app to view your purchased content!" },
   "app_badge_ios": { "other": "Download on the App Store" },
-  "app_badge_android": { "other": "Get it on Google Play" }
+  "app_badge_android": { "other": "Get it on Google Play" },
+  "awards_winner_of": { "other": "Ganador de" }
```

Comment:
looks like mx might be missing the other awards keys? in competition / nominated for